components/form/form.tsx

```diff
@@ -11,7 +11,7 @@ import { Description } from "../../ui/description/Description"
 import { motion } from "framer-motion"
 import { baseItemBottom } from "../animations/motion"
 import { useSpotify } from "../../common/hooks/useSpotify"
-import { CircularProgress } from "@material-ui/core"
```

Comment:
lol we use the material-ui circular progress at work, i really didn't catch & accidentally just used it, that's funny